packages/did-eth-registry/contracts/EIP1056Registry.sol

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.15;
+
+/**
+ * @title EIP1056Registry
+ * Specifies a did registry that implements EIP-1056
+ * https://eips.ethereum.org/EIPS/eip-1056
+ */
+interface EIP1056Registry {
+    event DIDOwnerChanged(address indexed identity, address owner, uint256 previousChange);
+
+    /**
+     * Delegation change event
+     */
+    event DIDDelegateChanged(
+        address indexed identity,
+        bytes32 delegateType,
+        address delegate,
+        uint256 validTo,
+        uint256 previousChange
+    );
+
+    /**
+     * Attribute change event
+     */
+    event DIDAttributeChanged(
+        address indexed identity,
+        bytes32 name,
+        bytes value,
+        uint256 validTo,
+        uint256 previousChange
+    );
+
+    /**
+     * Return the current owner of an identity.
+     * @param identity The identity to check.
+     * @return The address of the current owner.
+     */
+    function identityOwner(address identity) external view returns (address);
+
+    /**
+     * Return the current delegate of an identity for a specific delegate type.
+     * @param identity The identity to check.
+     * @param delegateType The type of the delegate.
+     * @param delegate The address of the delegate.
+     * @return bool The address of the current delegate.
+     */
```

Comment:
```suggestion
    /**
     * Determines if a specific address is currently listed as a delegate for a given identity and type of delegation.
     * @param identity - The identity to check.
     * @param delegateType - The type of the delegation being checked.
     * @param delegate - The address of the delegate.
     * @return bool - true if the delegation is currently valid, false otherwise.
     */
```